modules/deploy/pages/deployment-option/cloud/aws-privatelink.adoc

```diff
@@ -0,0 +1,394 @@
+
+. Check the service state by calling https://docs.api.redpanda.com/#get-/v1beta1/clusters/-id-[`GET /v1beta1/clusters/{id}`]. The `service_state` in the `private_link.status` response object needs to be `Available` for you to <<connect-to-privatelink-endpoint-service,connect to the service>>.
++
+[,bash]
+----
+curl -X GET \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer $AUTH_TOKEN" \
+    $PUBLIC_API_ENDPOINT/v1beta1/clusters/$CLUSTER_ID | jq '.private_link.status.aws | {service_name, service_state}'
+----
+
+== Connect to PrivateLink endpoint service
+
+=== Get cluster domain
+
+Get the domain (`cluster_domain`) of the cluster from the cluster details in the Redpanda Cloud UI.
+
+For example, if the Bootstrap server URL is: `seed-3da65a4a.cki01qgth38kk81ard3g.fmc.dev.cloud.redpanda.com:9092`, then `cluster_domain` is: `cki01qgth38kk81ard3g.fmc.dev.cloud.redpanda.com`.
+
+[,bash]
+----
+CLUSTER_DOMAIN=<cluster_domain>
+----
+
+=== Get name of PrivateLink endpoint service
+
+You need the service name to <<create-vpc-endpoint,create VPC private endpoints>>. Run the following command to get the service name:
+
+[,bash]
+----
+PL_SERVICE_NAME=`curl -X GET \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer $AUTH_TOKEN" \
+  $PUBLIC_API_ENDPOINT/v1beta1/clusters/$CLUSTER_ID | jq -r .private_link.status.aws.service_name`
+----
+
+=== Create client VPC
+
+If you are not using an existing VPC, you must create a new one. 
+
+The VPC region must be the same region where the Redpanda cluster is deployed. Run the following command to create the VPC:
+
+[,bash]
+----
+PROFILE=<specific-profile-from-your-credential-file>
+
+aws ec2 create-vpc --region $REGION --profile $PROFILE --cidr-block 10.0.0.0/20
+CLIENT_VPC_ID=<client_vpc_id>
+----
+
+You can also use an existing VPC. You need the VPC ID to <<modify-vpc-attributes,modify its attributes>>.
+
+=== Modify VPC attributes
+
+[,bash]
+----
+aws ec2 modify-vpc-attribute --region $REGION --profile $PROFILE --vpc-id $CLIENT_VPC_ID \
+    --enable-dns-hostnames "{\"Value\":true}"
+
+aws ec2 modify-vpc-attribute --region $REGION --profile $PROFILE --vpc-id $CLIENT_VPC_ID \
+    --enable-dns-support "{\"Value\":true}"
+----
+
+=== Create security group
+
+Run the following command. You need the security group ID `security_group_id` from the command output to <<add-security-group-rules,add security group rules>>.
+
+[,bash]
+----
+aws ec2 create-security-group --region $REGION --profile $PROFILE --vpc-id $CLIENT_VPC_ID \
+    --description "Redpanda endpoint service client security group" \
+    --group-name "${CLUSTER_ID}-sg" \
+SECURITY_GROUP_ID=<security_group_id>
+----
+
+=== Add security group rules
+
+This assumes that the Redpanda broker count is three. If you are not using three brokers, modify the example below:
```

Comment:
```suggestion
This setting is based on the assumption that the Redpanda broker count is three. If you are not using three brokers, modify the example below:
```

Reply:
Seems like the heading is not clearly describing the task. Is this section actually describing how to add the security group rules, or is it describing how to modify the number of brokers so that you can add security group rules?